crates/sargon/src/profile/v100/factors/factor_source_kind.rs

```diff
@@ -40,24 +28,24 @@ pub enum FactorSourceKind {
     #[serde(rename = "ledgerHQHardwareWallet")]
     LedgerHQHardwareWallet,
 
-    /// A user owned mnemonic (and optional BIP39 passphrase) user has to input when used,
-    /// e.g. during signing.
+    /// An Arculus card, in credit card size, communicating with host using NFC.
+    ///
+    /// For more info see [link]
     ///
     /// Attributes:
     ///  * Mine
     ///  * Off device
-    ///  * Hierarchical deterministic  (Mnemonic)
-    #[serde(rename = "offDeviceMnemonic")]
-    OffDeviceMnemonic,
-
-    /// A contact, friend, company, organization or otherwise third party the user trusts enough
-    /// to be given a recovery token user has minted and sent the this contact.
+    ///  * Hierarchical deterministic  (**Encrypted** mnemonic)\
+    ///  * Hardware (communicates with host using NFC)
     ///
-    /// Attributes:
-    ///  * **Not** mine
-    ///  * Off device
-    #[serde(rename = "trustedContact")]
-    TrustedContact,
+    /// [link]: https://www.getarculus.com/
+    #[serde(rename = "arculusCard")]
+    ArculusCard,
+
+    /// Input key material for mnemonic (and optional BIP39 passphrase).
+    /// TODO add explanation
```

Comment:
the attributes are:

```
    /// Attributes:
    ///  * Mine
    ///  * Off device
    ///  * Hierarchical deterministic (IKM -> HKDF -> Mnemonic)
```

very similar to OffDeviceMnemonic, it is essentially the same...